### PR TITLE
Fixes #15169: Reformat some cfengine classes bodies in ncf_lib.ncf

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15169

### CODE REVIEW

File: tree/20_cfe_basics/ncf_lib.cf [hunk 1]
```diff
@@ -862,30 +840,31 @@ body classes classes_generic_return_boolean_list_two(x, y, boolean_prefix, true_
     repaired_returncodes => { @{false_return_codes} };
 
     promise_repaired => { "promise_kept_$(x)", "$(x)_kept", "$(x)_ok", "$(x)_not_repaired", "$(x)_reached",
-                          "promise_kept_$(y)", "$(y)_kept", "$(y)_ok", "$(y)_not_repaired", "$(y)_reached", "${boolean_prefix}_false" };
-    repair_failed => { "repair_failed_$(x)", "$(x)_failed", "$(x)_not_ok", "$(x)_not_kept", "$(x)_not_repaired", "$(x)_reached", "${x}_error",
```
Comment (amousset) [inline tree/20_cfe_basics/ncf_lib.cf]: why removing these?

**Comment by Fdall** [comment]
PR updated with a new commit

File: tree/20_cfe_basics/ncf_lib.cf (hunk 1 above)
Comment (Fdall) [reply tree/20_cfe_basics/ncf_lib.cf]: Wrong copy/paste, I forgot the inherit_from part

File: tree/20_cfe_basics/ncf_lib.cf [hunk 2]
```diff
@@ -861,31 +839,34 @@ body classes classes_generic_return_boolean_list_two(x, y, boolean_prefix, true_
     kept_returncodes => { @{true_return_codes} };
     repaired_returncodes => { @{false_return_codes} };
 
+    inherit_from => classes_generic_two("${x}", "${y}");
+
     promise_repaired => { "promise_kept_$(x)", "$(x)_kept", "$(x)_ok", "$(x)_not_repaired", "$(x)_reached",
```
Comment (amousset) [inline tree/20_cfe_basics/ncf_lib.cf]: why keeping this one?

Comment (Fdall) [reply tree/20_cfe_basics/ncf_lib.cf]: We need the "${boolean_prefix}_false" one, and I am pretty sure that cfengine does not allow merging of body attributes but override only.

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**